cmd/juju/model/destroy.go

```diff
@@ -338,9 +338,13 @@ func newTimedModelStatus(ctx *cmd.Context, api DestroyModelAPI, tag names.ModelT
 			return nil
 		}
 		if status[0].Error != nil {
-			// This most likely occurred because a model was
-			// destroyed half-way through the call.
-			ctx.Infof("Could not get the model status from the API: %v.", status[0].Error)
+			// No need to give the user a warning that the model they asked
+			// to destroy is no longer there.
+			if errors.IsNotFound(status[0].Error) || params.IsCodeNotFound(status[0].Error) {
+				ctx.Infof("Model destroyed.")
```

Comment:
If we're adding logging for this one, we should probably do it when checking the immedate error result of ModelStatus, too.